src/Controller/RegistrationController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use App\Repository\UserRepository;
+use App\Form\RegistrationFormType;
+use App\Form\UserType;
+use App\Security\LoginFormAuthenticator;
+use App\Security\UserAuthenticationAuthenticator;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Symfony\Component\Security\Guard\GuardAuthenticatorHandler;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+/**
+ * @Route("/install")
```

Comment:
It's not mandatory to have a @Route annotation for controller classes.
And for a class with only one controller, it could be easier to read if the full route was in the method's annotation